packages/create-qwik/create-interactive.ts

```diff
@@ -114,24 +124,25 @@ export async function runCreateInteractiveCli() {
 
   if (gitInitAnswer) {
     if (fs.existsSync(join(outDir, '.git'))) {
-      console.log(`Git has already been initialized`);
-      console.log(``);
-      return;
```

Comment:
atm we stop the whole CLI flow if this situation occurs which is from my point of view suboptimal.